cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaConnectConfigurationBuilderTest.java

```diff
@@ -424,7 +438,7 @@ public void testWithUserProvidedAndDefaultConfigurations() {
         KafkaConnectConfiguration configurations = new KafkaConnectConfiguration(Reconciliation.DUMMY_RECONCILIATION, userConfiguration.entrySet());
 
         String configuration = new KafkaConnectConfigurationBuilder(BOOTSTRAP_SERVERS)
-                .withUserConfigurations(configurations)
+                .withUserConfiguration(configurations, true, true)
```

Comment:
This should be all false to preserve the existing behavior.

Reply:
Done